### ISSUE TITLE
Can we remove/change this? I think it could scare people away from attempting DIY Nightscout.

### ISSUE THREAD

**Issue by maryannep** [nightscout/nightscout.github.io#134]
As a user of Nightscout for 4 years, (thank you so much developers) I'm concerned the following section of the Nightscout docs might lead people to unnecessarily believe setting up a Nightscout site using the 'DIY' option is going to be arduous, painful and require them to do hours of ongoing work for maintenance. I've found this isn't the case at all (at the user end anyway, currently. I do realise there must have been a incredible amount of work done to create Nightscout in the first place and there is still a lot of work done behind the scenes by developers & volunteers online in community forums.) 

Could/should the section below and attached from nightscout.github.io be deleted?

"There aint no such thing as a free lunch. You might not pay for service, but also this means you'll be spending a lot of hours on this.
![Screen Shot 2021-11-13 at 4 12 00 pm](https://user-images.githubusercontent.com/42899893/141609343-2e7c9697-290e-43f2-ac67-3f698c66928e.png)
"

**Comment by psonnera** [comment]
This has been discussed here https://github.com/nightscout/nightscout.github.io/issues/8 and I understand it might scare some people.

**Comment by bewest** [comment]
I think I agree the wording could be reworked to be more welcoming for all.  We understand different people have different needs.  Some want a frugal, low risk, low cost option.  Others want a coin-operated service.  There's no need to be acrimonious to either groups.  We recognize nice both sets of need without passing judgement.
https://nightscout.github.io/#what-are-the-values-of-nightscout

Feel free to provide suggest edits to make the values of our community clearer, as well as bring the welcoming message more in line with our values.

**Comment by maryannep** [comment]
Hi Ben, it's always good to hear from you :-) Sorry for any confusion. I was simply suggesting removing the whole section in the blue box because I don't think "you'll be spending lots of hours on this" is true. 

I'm a T1Pal subscriber who loves your secret share links functionality. I also built my own Heroku site a year ago to test out just how much maintenance a free tier would involve because I started noticing online comments about it taking 8 hours a year. I found it was very little. My initial build was done by my partner, who set me up with the $7 per month dyno hours option and I was interested to read Sulka's comments on the value of that too.
